lib/container_orchestrator/object_definition.rb

```diff
@@ -2,18 +2,34 @@ class ContainerOrchestrator
   module ObjectDefinition
     private
 
+    def common_labels
+      {:app => app_name}.merge(orchestrated_by_label)
+    end
+
+    def orchestrated_by_label
+      {:"#{app_name}-orchestrated-by" => pod_name}
+    end
+
+    def orchestrated_by_selector
+      "#{app_name}-orchestrated-by=#{pod_name}"
+    end
+
+    def app_name_selector
+      "app=#{app_name}"
+    end
```

Comment:
Minor, but these helper methods should probably go down where app_name is defined for consistency (the defintions are at the top and helper methods at the bottom from what I can tell).